src/Runtime/StudioGateway/src/StudioGateway.Api/Endpoints/Internal/FluxWebhookEndpoints.cs

```diff
@@ -1,11 +1,15 @@
-using StudioGateway.Api.Authentication;
+using StudioGateway.Api.Clients.Designer.Contracts;
+using StudioGateway.Api.Clients.K8s;
 using StudioGateway.Api.Endpoints.Internal.Contracts;
 using StudioGateway.Api.Hosting;
+using HelmReleaseNameParser = StudioGateway.Api.Clients.K8s.HelmReleaseNameParser;
 
 namespace StudioGateway.Api.Endpoints.Internal;
 
 internal static class FluxWebhookEndpoints
 {
+    private sealed record HelmReleaseInfo(string Org, string App, string SourceEnvironment, string? BuildId);
+
     public static WebApplication MapFluxWebhookEndpoint(this WebApplication app)
     {
         app.MapPost("/api/v1/flux/webhook", HandleFluxWebhook)
```

Comment:
What happens if this endpoint fails?

Reply:
Flux will retry few times, and designer client have retry itself, but if all fails event wont be send to designer